tests/__xlsx__/xlsx_demo_data/docProps/app.xml

```diff
@@ -9,7 +9,7 @@
         <vt:lpstr>Worksheets</vt:lpstr>
       </vt:variant>
       <vt:variant>
-        <vt:i4>10</vt:i4>
```

Comment:
lots of noise in the xlsx demo data because 

1) I removed a string + a format, so all the indexes of styles/shared string moved around
2) Excel dark magic